tests/search/tfidf_test.cpp

```diff
@@ -1242,7 +1248,7 @@ TEST_P(tfidf_test, test_order) {
   *query.mutable_field() = "field";
 
   iresearch::order ord;
-  ord.add<iresearch::tfidf_sort>(true);
+  ord.add<iresearch::tfidf_sort>(true, false, true);
```

Comment:
ASSERT use_boost_as_score() is true here ? And maybe add test where setting boost_as_score = true affects results ?